README.adoc

```diff
@@ -1,68 +1,205 @@
-If you do navigate to a `file://` URL from a link, simply highlight the address bar and press "Enter" to force the extension to load. There's an open issue in the https://bugzilla.mozilla.org/show_bug.cgi?id=1266960[Firefox bug tracker] to fix this limitation.
-====
+After revewing and changing the options below, continue to <<next-steps>>.
 
-== Installation
+Asciidoctor{nbsp}options::
++
+* *Custom attributes*: None (default)
++
+A string of space-separated key/value attribute pairs. For example:
++
+[,txt]
+----
+toc=left experimental icons=font source-highlighter=highlight.js
+----
++	
+https://docs.asciidoctor.org/asciidoc/latest/attributes/document-attributes[Learn more] about attributes via docs.asciidoctor.org.
 
-Loading your local copy of the extension on Chrome (or Chromium), Firefox or Opera is super easy!
+* *Safe mode*: Safe (default)
++
+Controls features related to security. 
+https://docs.asciidoctor.org/asciidoctorj/latest/safe-mode-and-filesystem-access[Learn more] about these values via docs.asciidoctor.org.
 
-=== Chrome, Chromium, Brave or Opera
+
+Files extension:: 
++
+* *Preview files with `txt` extension*: Disabled (default)
++
+Useful when viewing raw text files with AsciiDoc content from hosted git providers or your local machine.
+
+Poll for changes::
++
+* *To files loaded from URLs*: 2 seconds (default)
+* *To local files*: 2 seconds (default)
+
+Diagrams{nbsp}extension::
++
+* *Enable the diagrams extensions*: Disabled (default)
+* *Server URL*: `https://kroki.io` (default)
++
+.Information leaving your computer
+IMPORTANT: When you enable the diagrams extension without changing the default *server URL*, you agree to the diagrams extension sending your diagrams (https & base64 encoded) to the free public cloud instance https://kroki.io for conversion.
++
+You or your enterprise may install a Kroki server on your own infrastructure using either containers or a local build and install.
+Once you've installed the local or enterprise Kroki server, make sure to use the correct server URL to point to your instance.
++
+.Benefits of a local Kroki.io instance
+TIP: For example, you are able to set standard technical diagram themes for consistency across your organization or team without any changes to the diagrams themselves.
+
+Additional{nbsp}resources::
++
+* *Theme/Stylesheet*: Default
+* *Add a stylesheet*: None (default)
+
+Javascript::
++
+* *Load the Javascript*: Before (default)
++
+After or before the document has been rendered.
+
+== Next steps
+You can now open local or remote *.adoc*, *.ad*, *.asc*, *.asciidoc* (and optionally txt) files in your Chromium browser to view the rendered AsciiDoc.
+
+You can print to PDF using the browser print function.
+
+You can change the theme (perhaps to dark) by <<configure-extension-options>>. 
+
+The Extension can be toggled between active (Red) and inactive (Gray) by clicking its icon in the extension bar.
```

Comment:
Instead of "red" we might want to say between active (colored) and inactive (greyed-out) ? Since the color "red" is often used as a "stop"/"error" state.

Reply:
I trialed so many ways to describe this. Thanks for being succinct!